decksite/controllers/api.py

```diff
@@ -143,6 +157,41 @@ def decks_api() -> Response:
     resp.set_cookie('page_size', str(page_size))
     return resp
 
+@APP.api.route('/scraper/')
+class Scraper(Resource):
+    @APP.api.marshal_with(MULTIPLE_DECKS)
+    def get(self) -> Dict[str, Any]:
+        """
+        Grab a slice of finished sorted decks last updated after a certain point.
+        Input:
+            {
+                'sortBy': <str>,
+                'sortOrder': <'ASC'|'DESC'>,
+                'page': <int>,
+                'pageSize': <int>,
+                'lastTimestamp': <int>,
```

Comment:
I like `since` or `sinceTimestamp` if you must for this.

`/decks/updated/?since=9393938229`

Nice.

Reply:
sure ill do that